ingest/Snakefile

```diff
@@ -34,7 +36,9 @@ dataset_server_map = {}
 dataset_name_map = {}
 
 if SEGMENTED:
-    GROUPS_OVERRIDE_JSON = config.get("grouping_override", None)  # JSON map from group to segments' insdcAccessionFull
+    GROUPS_OVERRIDE_JSON = config.get(
```

Comment:
can you separate out the formatting changes? I.e. run `ruff format` on everything first, make that pr and then make this one?

Reply:
sorry this is snakemake, so `snakefmt` it